github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -698,7 +698,7 @@ periodics:
       - /bin/bash
       - -c
       - ../project-infra/hack/git-pr.sh -c "cd ../kubevirt && make rpm-deps" -b bump-rpm-dependencies
-        -p ../kubevirt -T main -R
+        -p ../kubevirt -T main -R -L skip-review
```

Comment:
/hold

add release-blocker label in this PR creation

Reply:
release-blocker plugin would undo the labelling [1] from adding it during the PR creation. I think we should merge this PR first and then handle the addition of the label via comment in another PR. @fossedihelm WDYT?

[1]: https://github.com/kubevirt/project-infra/blob/main/external-plugins/release-blocker/server.go#L237

Reply:
Makes sense, thanks